src/Hl7.Fhir.Specification/Specification/Terminology/ITerminologyService.cs

```diff
@@ -7,11 +7,75 @@
  */
 
 using Hl7.Fhir.Model;
+using System;
 
 namespace Hl7.Fhir.Specification.Terminology
 {
     public interface ITerminologyService
```

Comment:
All these new methods accept the "untyped" Parameters as a parameter, while we also have the nicely "typed" equivalents (like `ExpandParameters`).  Can we also provide extension methods on top of this interface that accept the correct "typed" parameter, and just call these methods  (calling the correct `Build()` to turn them into an instance of `Parameters`)?

Reply:
Like this?

```c#
        public static Parameters Subsumes(this ITerminologyService srv, SubsumesParameters parameters)
        {
            return srv.Subsumes(parameters.Build());
        }
```